packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts

```diff
@@ -344,10 +346,13 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
               },
             },
           },
+          // If hasFaceMatchedInOval is true, then move to flashFreshnessColors
+          // and pause for one second to show "Hold still" text
+          // If not, move back to ovalMatching and re-evaluate match state
```

Comment:
Think this comment is incorrect, suggest
```suggestion
          // If `hasFaceMatchedInOval` is true, then move to `delayBeforeFlash`, which pauses 
          // for one second to show "Hold still" text before moving to `flashFreshnessColors`.
          // If not, move back to ovalMatching and re-evaluate match state
```